templates/security-headers.conf

```diff
@@ -0,0 +1,2 @@
+#enable X-Frame Options header to indicate whether or not a browser should be allowed to render a page in a <frame>, <iframe>, <embed> or <object>
+add_header X-Frame-Options "SAMEORIGIN";
```

Comment:
i would make SAMEORIGIN also configurable with an env var, so you can for example change it to DENY, otherwise cannot be used